docs/source/terms_and_model.rst

```diff
@@ -393,30 +387,44 @@ The same APOE-ε1 Haplotype with referenced Alleles::
     }
 
 The GA4GH computed identifier for these Haplotypes is
-`ga4gh:VH.NAVnEuaP9gf41OxnPM56XxWQfdFNcUxJ`, regardless of the whether
-the Variation objects are inlined or referenced, and regardless of
-order. See :ref:`computed-identifiers` for more information.
-
-
+`ga4gh:VH.NAVnEuaP9gf41OxnPM56XxWQfdFNcUxJ`, regardless
+of whether the Variation objects are inlined or
+referenced, and regardless of order. See
+:ref:`computed-identifiers` for more information.
 
 .. _SystemicVariation:
 
 Systemic Variation
 ##################
 
+Systemic Variation is a :ref:`Variation` of multiple
+molecules in the context of a system, e.g. a genome,
+sample, or homologous chromosomes.
+
+.. _Abundance:
+
+Abundance
```

Comment:
There is good semantic value in doing so, at least here. I think we should keep Abundance. Similar to keeping SequenceInterval when we only had the SimpleInterval class (since we already had a good idea that nested interval was coming).